crates/forge/src/run_tests/package.rs

```diff
@@ -39,6 +39,7 @@ pub struct RunForPackageArgs {
     pub forge_config: Arc<ForgeConfig>,
     pub fork_targets: Vec<ForkTarget>,
     pub package_name: String,
+    pub exclude_filter: Option<NameFilter>
```

Comment:
It's no use in storing both `tests_filter` and `exclude_filter` since the former includes the latter. Also, it's only used in an obsolete line [here](https://github.com/foundry-rs/starknet-foundry/blob/dfd78a956ae0acd47705b29f5e07aec9d0ebc7c6/crates/forge/src/run_tests/package.rs#L140), which I pointed out in another comment.